pkg/actors/actor_test.go

```diff
@@ -33,39 +33,109 @@ func TestConcurrencyLocks(t *testing.T) {
 	}()
 
 	time.Sleep(10 * time.Millisecond)
+	assert.Equal(t, int32(2), testActor.pendingActorCalls)
 	assert.True(t, testActor.isBusy())
 	assert.Equal(t, firstLockTime, testActor.lastUsedTime)
 
 	// unlock the first lock
-	testActor.unLock()
+	testActor.unlock()
 
 	time.Sleep(10 * time.Millisecond)
+	assert.Equal(t, int32(1), testActor.pendingActorCalls)
 	assert.True(t, testActor.isBusy())
 	assert.NotEqual(t, firstLockTime, testActor.lastUsedTime)
```

Comment:
check `assert.True(t, testActor.After(firstLockTime.Add(10*time.Millisecond)))`

Reply:
After returns channel :) anyhow, I change to check the duration.